data/editor.config

```diff
@@ -410,6 +410,10 @@
         <include group="poi" />
         <include field="operator" />
       </type>
+      <type id="boundary-marker">
+        <include field="name" />
+        <include field="ref" />
```

Comment:
unfortunately "ref" is not supported by the editor yet

Reply:
Thanks, fixed